<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestDlg</class>
 <widget class="QDialog" name="TestDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>测试窗体</string>
  </property>
  <widget class="QLineEdit" name="lineEditInFile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>261</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonInFile">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOutFile">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOutFile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>261</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCanCel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
